### DISCUSSION TITLE
Blacktop Blitz Discussion Forum

### DISCUSSION THREAD

**Discussion by wkoverfield** [wkoverfield/blacktop-blitz#2] [Announcements]
Welcome to the Blacktop Blitz discussion forum! This space is dedicated to all things related to Blacktop Blitz, a React application designed to help NBA 2K players quickly generate and draft teams for the Blacktop game mode. 

Feel free to report bugs, share feedback, or suggest new features.

**Comment by DirtTheElement** [comment]
Hi I really love your website, I was wondering if you could make these but of past 2Ks. Also are you planning on updating the website for 2K25?

**Reply by wkoverfield** [reply]
Thanks so much, I'm glad you like the website! I'll look into maybe being able to do past 2ks but I plan to release an update for 2k25.

**Reply by wkoverfield** [reply]
Hey, I wanted to say I updated it, and in about a week or so I'm going to have another big update!

**Reply by DirtTheElement** [reply]
Hey, I loved your new update. Me and my friends still use this to this day, excited to see the update for next 2k. Keep up the good work!!

**Reply by wkoverfield** [reply]
Hey, thank you, this honestly made my day! It's awesome to hear you and your friends are still playing after all this time.

And I haven't forgotten your original ask about past 2Ks. I've figured out a way to pull old rosters, and it's on my list of things on the way soon!
